lib/config/configuration.go

```diff
@@ -681,18 +681,22 @@ func applyLogConfig(loggerConfig Log, cfg *servicecfg.Config) error {
 		w = os.Stdout
 		cfg.Console = io.Discard // disable console printing
 	case teleport.Syslog:
-		// TODO(tross): add slog support for syslog.
-		hook, err := utils.CreateSyslogHook()
+		w = os.Stderr
+		sw, err := utils.NewSysLogWriter()
```

Comment:
```suggestion
		sw, err := utils.NewSyslogWriter()
```

Nit: it's Syslog, not SysLog.